inspirehep/records/cli.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import json
+import os
+
+import click
+import requests
+from flask.cli import with_appcontext
+from invenio_db import db
+
+from inspirehep.records.api import AuthorsRecord, LiteratureRecord
+
+
+def _create_record(data):
+    control_number = data["control_number"]
+    click.echo(f"Creating record {control_number}.")
+    # ``earliest_date``` is not part of the schema and fails on
+    # validation. We are adding it in the serializers, hence
+    # it's part of our API responses.
+    data.pop("earliest_date", None)
+
+    # FIXME: move the type creation logic in ``InspireRecord.create``
```

Comment:
just not to forget, do we have a JIRA issue for this?

Reply:
YEP we have INSPIR-2210